packages/mobile/src/components/navigation-container/NavigationContainer.tsx

```diff
@@ -258,6 +266,13 @@ const NavigationContainer = (props: NavigationContainerProps) => {
         })
       }
 
+      const { query } = queryString.parseUrl(path)
+      const { login, warning } = query
+
+      if (login && warning) {
```

Comment:
what's warning? how do we know it's the right one?

Reply:
so looking at the index.html, there is a check for warning as well as login, to indicate that this is a "reset-password" link. if just login is there, then it's a "log-in" and go to the feed. its the "warning" part that means "open reset-password"